src/server/db/authors-dbConfiguration.js

```diff
@@ -0,0 +1,50 @@
+/*
+For copyright information, see the AUTHORS.md file in the docs directory of this distribution and at
+https://github.com/fluid-project/sjrk-story-telling/blob/main/docs/AUTHORS.md
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.txt
+*/
+
+// These eslint directives prevent the linter from complaining about the use of
+// globals or arguments that will be prevent in CouchDB design doc functions
+// such as views or validate_doc_update
+
+/* global emit */
+
+"use strict";
+
+require("infusion");
+require("kettle");
+
+var sjrk = fluid.registerNamespace("sjrk");
+require("fluid-couch-config");
+
+// sets up the Storytelling Tool database using fluid-couch-config
+fluid.defaults("sjrk.storyTelling.server.authorsDb", {
+    gradeNames: ["fluid.couchConfig.pipeline.retrying"],
+    couchOptions: {
+        dbName: "authors"
```

Comment:
Just curious, is "authors" a separated database that only contains the "user" document type? What's the reason to decide on using a separate database rather than making use of the existing "stories" database with a new "user" document type?

With the current logic, using 2 databases works the same way. But it feels strange using multiple databases to support an app, which is unnecessary.

Reply:
That the user accounts are managed through fluid-express-users library, but I think it could actually be in the same database as stories. I'll look into that.

Reply:
I looked into this when I first saw @jobara's choice. Turns out using separate databases is actually preferable, on account of the way index building works in CouchDB, though the performance gains to us will be minimal. Here's a very interesting thread on the topic, I recommend reading the answers other than the one the OP chose: https://stackoverflow.com/questions/10503089/couchdb-multiple-databases

Reply:
@BlueSlug that's a very interesting point. Unfortunately for me I already combined them. @cindyli and @BlueSlug let me know if you think we should split them back out.

Reply:
It's great to know separating data into multiple databases helps with the CouchDB performance. Thanks for the good thread, @BlueSlug. After reading the thread, I'm fine with either.

If @jobara has already combined it, I would prefer leave as combined. As @BlueSlug said, the performance gain for SJRK will be minimal. Moreover, using one database simplifies the code and is easier for backup and maintenance.